### ISSUE TITLE
Music / Sound effects

### ISSUE THREAD

**Issue by mybandequalsme** [SneakyTactician/MagicalLife#47]
Hi guys!  I replied to your request for a composer on Reddit.  My name is Sam Hammoudeh and I compose and produce music for games and short films, as well as produce edm.

**Comment by SneakyTactician** [comment]
Hi Sam!

Since you have commented here, I know you have seen all of the information in this repository. What do you think? Are you interested in helping out?

**Comment by SneakyTactician** [comment]
Lol, I meant the "Vision"/ end goal. Do you have any experience with FMOD?

**Comment by mybandequalsme** [comment]
Ha, yes the vision looks good.  I don't, but I can learn it.

**Comment by SneakyTactician** [comment]
Currently my other composer is telling me that I need to give him something to go off of to make music, and that this project is really ready for a composer yet. Mind if I send you a message when the project is ready for composers?

**Comment by mybandequalsme** [comment]
Sounds good.  Just let me know.

**Comment by mybandequalsme** [comment]
Hi Sneaky.  Any word on the project?  Is it ready for a composer yet?

**Comment by SneakyTactician** [comment]
I'm afraid that it is not at this time. Since we talked last, trees and rock have been added. Chopping/mining of rocks is done. Crafting of wood logs into planks is a few days of work. I'm not sure if we're ready yet. Maybe when enemies are added in and combat is ready, then we would be ready for some combat music and the like.

**Comment by a2937** [comment]
Do we have the ability to play sounds actually?

**Comment by SneakyTactician** [comment]
Yes, playing sounds is supported. I believe that there is a music player class somewhere too, that can be used to play music.

Thomas

________________________________
